src/recommender.py

```diff
@@ -240,10 +250,19 @@ def _send_action(self, speaker_id, action):
             #empathid of the recommendation
             empathid = '999|' + time2
 
-            url = phone_url + '/?q={%22id%22:%22' + str(speaker_id) + '%22,%22c%22:%22startsurvey%22,%22suid%22:%22' + \
-              survey_id[action] + '%22,%22server%22:%22' + server_url + '%22,%22androidid%22:%22' + \
-              androidid + '%22,%22empathid%22:%22' + empathid + \
-              '%22,%22alarm%22:%22' + alarm + '%22}'
+            url_dict = {
+              "id": str(speaker_id),
```

Comment:
one thing I think we should agree on next is some coding conventions, it is better for us to follow a standard so the code looks more "unified" and it's easier for us to touch each other's codes

like the quotes here, since python allows both single and double quote, we should decide which one to use. how about single quote `'`, which is more common is python community?